acra/configuring-maintaining/optimisations/acrastructs_vs_acrablocks.md

```diff
@@ -0,0 +1,44 @@
+---
+title: AcraStructs vs AcraBlocks
+weight: 3
+---
+
+# AcraStructs vs AcraBlocks
+
+There are two different "crypto containers" AcraServer can use to store encrypted value in.
+Then, these containers are stored in database and decrypted back by AcraServer when needed.
+
+`AcraStruct` is the main crypto container for the client-side encryption.
+Applications can encrypt with ephemeral asymmetric keys on their side with `AcraStruct`s,
+but cannot do this with `AcraBlock`s because of symmetric key encryption.
+You can also configure AcraServer to re-encrypt `AcraStruct`s (that were generated on app side)
+on AcraServer side, this way data will definitely be encrypted while transmitted between
+some client app and AcraServer.
+
+### Encryption with a key
+
```

Comment:
I don't really like this header (it's don't say anything to me), but I don't have any real suggestions either. IDK, “Key wrapping”? Something that *actually* relates to the matter discussed in this section. “Encryption with a key” is not it. It's obvious that you need a key for encryption. But the section is not about that, it's about how both containers have a DEK but they deal with KEK differently.